src/subscription_manager/identity.py

```diff
@@ -169,3 +169,6 @@ def __str__(self):
         return "<%s, name=%s, uuid=%s, consumer=%s>" % \
                 (self.__class__.__name__,
                 self.name, self.uuid, self.consumer)
+
+    def log_str(self):
```

Comment:
Not a huge deal or anything, but the existing `__str__` method looks more like a `__repr__` to me.  Maybe make that switch and then set `log_str` to be `__str__`?